contracts/DaiHrd.sol

```diff
@@ -431,8 +431,8 @@ contract ERC777 is RuntimeConstants, Context, IERC777, IERC20 {
 
 contract MakerFunctions {
 	uint constant ONE = 10 ** 27;
-	function rmul(uint x, uint y) internal pure returns (uint z) {
-		z = mul(x, y) / ONE;
+	function safeMul27(uint x, uint y) internal pure returns (uint z) {
```

Comment:
These are more sane names, might be worthwhile to leave a comment about what original name they come from?